app/components/primer/alpha/tool_tip.ts

```diff
@@ -18,30 +17,34 @@ const DIRECTION_CLASSES = [
   'tooltip-sw'
 ]
 
+const tooltips = new Set<ToolTipElement>()
 class ToolTipElement extends HTMLElement {
   styles() {
     return `
       :host {
         position: absolute;
         z-index: 1000000;
-        padding: .5em .75em;
+        padding: .5em .75em !important;
         font: normal normal 11px/1.5 -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji";
         -webkit-font-smoothing: subpixel-antialiased;
-        color: var(--color-fg-on-emphasis);
+        color: var(--color-fg-on-emphasis) !important;
```

Comment:
@langermank I didn't think about it before, but these old variables don't exist in the `.pcss` pipeline.